src/Mayaqua/Network.c

```diff
@@ -16615,7 +16615,6 @@ TOKEN_LIST *GetCipherList()
 	}
 
 	sk_SSL_CIPHER_free(sk);
-	FreeSSLCtx(ctx);
 	SSL_free(ssl);

```

Comment:
should we also assign NULL inside FreeSSLCtx ? there are other usages of that function as well

Reply:
It would be good practice.